components/Notifications/NotificationEntry.js

```diff
@@ -67,14 +72,39 @@ const NotificationEntry = (props) => {
     return [];
   };
 
-  const formatBody = (body) => {
+  const formatBody = (notification, data) => {
+    const { notification_type } = data;
+    const { body } = notification;
     const onClick = (e) => {
       e.stopPropagation();
       markAsRead(data);
       props.closeMenu();
     };
 
-    if (body == null) {
+    if (notification_type === "ACCOUNT_VERIFIED") {
+      return "Congratulations! Your account has been verified by the ResearchHub team. ";
+    } else if (notification_type === "PAPER_CLAIMED") {
+      const unifiedDocument = data?.unified_document
+        ? parseUnifiedDocument(data.unified_document)
+        : null;
+      const url = getUrlToUniDoc(unifiedDocument);
+
+      return (
+        <>
+          <span>
+            Congratulations! You have been verified as an author of {` `}
+          </span>
+          {url && (
+            <HyperLink
+              link={{ href: url }}
+              onClick={onClick}
+              style={styles.link}
+              text={unifiedDocument.document.paperTitle}
+            />
+          )}
+        </>
+      );
+    } else if (body == null) {
```

Comment:
Good catch. I ended up not doing that for a couple of reasons:

1. I don't like the notification mechanism in the backend. We currently save the message and formatting in the DB which makes it harder to update in the future. The backend should not dictate how a notification is rendered on the FE. The API should just return data and let the app decide how to render it.
2. The particular notification I am rendering breaks the current convention because there is no "triggering user" unlike other notifications (which means no avatar being rendered)

Even though we are breaking consistency, I think it is okay to break out of the flow given that things are still backward compatible.